charger/easee.go

```diff
@@ -71,10 +72,11 @@ func init() {
 // NewEaseeFromConfig creates a go-e charger from generic config
 func NewEaseeFromConfig(other map[string]interface{}) (api.Charger, error) {
 	cc := struct {
-		User     string
-		Password string
-		Charger  string
-		Timeout  time.Duration
+		User      string
+		Password  string
+		Charger   string
+		Timeout   time.Duration
+		authorize bool
```

Comment:
If all strings starts with capital letter wouldn’t it be better to continue that with Authorize

Reply:
agreed